tensorflow_decision_forests/keras/core.py

```diff
@@ -301,29 +301,28 @@ class CoreModel(InferenceCoreModel):
     learner: The learning algorithm used to train the model. Possible values
       include (but at not limited to) "LEARNER_*".
     learner_params: Hyper-parameters for the learner. The list of available
-      hyper-parameters is available at
-      https://github.com/google/yggdrasil-decision-forests/blob/main/documentation/learners.md.
+      hyper-parameters is available at [Learners](https://github.com/google/yggdrasil-decision-forests/blob/main/documentation/learners.md).
```

Comment:
Please revert this, sometimes we prefer a raw link

Reply:
Okay. I've reverted the link back to its raw form, can you check it again?

Reply:
Thank you, I'll start our internal review process (may take a bit of time, so please be patient)

Reply:
Okay. Thanks for your time.